src/shared-components/container/style.ts

```diff
@@ -19,14 +19,16 @@ const messageStyle = (theme: ThemeType) => `
   box-shadow: ${theme.BOX_SHADOWS.clickable};
 `;
 
-export type ContainerType = 'clickable' | 'message';
+export type ContainerType = 'clickable' | 'message' | 'none';
 
 const getContainerTypeStyles = (theme: ThemeType, type?: ContainerType) => {
   switch (type) {
     case 'message':
       return messageStyle(theme);
     case 'clickable':
       return clickableStyle(theme);
+    case 'none':
```

Comment:
Explicitly encoding the non-message and non-clickable style in a type, in the event that we continue building on this and the behavior of the four cases change.